Modules/gcmodule.c

```diff
@@ -1754,6 +1836,7 @@ PyVarObject *
 _PyObject_GC_Resize(PyVarObject *op, Py_ssize_t nitems)
 {
     const size_t basicsize = _PyObject_VAR_SIZE(Py_TYPE(op), nitems);
+    assert(!_PyObject_GC_IS_TRACKED(op));
```

Comment:
Is it the same as `assert(!IS_TRACKED(op))` below?

Reply:
Yes, I added below macro in master branch in another pull request.
assert above can be removed.